docroot/sites/settings/cgov_caching.settings.php

```diff
@@ -7,14 +7,15 @@
  * @see https://docs.acquia.com/site-factory/tiers/paas/workflow/hooks
  */
 
+ if (!function_exists('settings_get_environment')) {
```

Comment:
Since we know that `settings_get_environment` is only set in `cgov_settings_utilities.inc`, is the `function_exists` check really necessary? The `require_once` should serve to prevent it being loaded twice.

Reply:
That would be a safe change.